docs/tutorials/jokes.md

```diff
@@ -6351,10 +6351,118 @@ This will build the docker image and deploy it on Fly in the region you selected
+Phew! And there we have it. If you made it through this whole thing then I'm really impressed ([tweet your success][tweet-your-success])! There's a lot to Remix and we've only gotten you started. Good luck on the rest of your Remix journey!
+
+[remix-jokes]: https://remix-jokes.lol
+[mdn]: https://developer.mozilla.org/en-US/
+[prisma]: https://www.prisma.io/
+[code-sandbox]: https://codesandbox.com/
+[node-js]: https://nodejs.org
+[npm]: https://www.npmjs.com
+[vs-code]: https://code.visualstudio.com/
+[fly-io]: https://fly.io
+[java-script-to-know-for-react]: https://kentcdodds.com/blog/javascript-to-know-for-react
+[the-beginner-s-guide-to-react]: https://kcd.im/beginner-react
+[the-http-api]: https://developer.mozilla.org/en-US/docs/Web/HTTP
+[the-basic-example]: https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/basic
+[express]: https://expressjs.com/
+[hydrate]: https://reactjs.org/docs/react-dom.html#hydrate
+[http-localhost-3000]: http://localhost:3000
+[bare-bones-hello-world-app]: /jokes-tutorial/img/bare-bones.png
+[remix-config-js]: ../api/conventions#remixconfigjs
+[route-module]: ../api/conventions#route-module-api
+[the-route-filename-convention]: ../api/conventions#route-filenames
+[react-router]: https://reactrouter.com/
+[a-greeting-from-the-index-route]: /jokes-tutorial/img/index-route-greeting.png
+[jokes]: http://localhost:3000/jokes
+[a-random-joke-on-the-jokes-page-i-was-wondering-why-the-frisbee-was-getting-bigger-then-it-hit-me]: /jokes-tutorial/img/random-joke.png
+[jokes-new]: http://localhost:3000/jokes/new
+[a-new-joke-form]: /jokes-tutorial/img/new-joke.png
+[jokes-anything-you-want]: http://localhost:3000/jokes/hippos
+[a-new-joke-form-2]: /jokes-tutorial/img/param-route.png
+[links]: ../api/conventions#links
+[links-2]: ../api/remix#link
+[http-localhost-3000-2]: http://localhost:3000/
+[the-homepage-with-a-purple-gradient-background-and-white-text-with-the-words-hello-index-route]: /jokes-tutorial/img/homepage-styles.png
+[the-jokes-page-with-no-background-gradient]: /jokes-tutorial/img/jokes-no-styles.png
+[check-out-the-mdn-page-for-link]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link
+[styled-components]: https://www.styled-components.com/
+[tailwind]: https://tailwindcss.com/
+[firebase]: https://firebase.google.com/
+[supabase]: https://supabase.com/
+[airtable]: https://www.airtable.com/
+[hasura]: https://hasura.io/
+[google-spreadsheets]: https://www.google.com/sheets/about/
+[cloudflare-workers-kv]: https://www.cloudflare.com/products/workers-kv/
+[fauna]: https://fauna.com/features
+[postgre-sql]: https://www.postgresql.org/
+[a-vs-code-extension]: https://marketplace.visualstudio.com/items?itemName=Prisma.prisma
+[sq-lite]: https://sqlite.org/index.html
+[prisma-2]: https://www.prisma.io
```

Comment:
Same for `prisma`

Reply:
we're now removing trailing slashes in the plugin 😄